modules/space/rendering/renderableeclipsecone.cpp

```diff
@@ -45,91 +45,85 @@ namespace {
     };
 
     constexpr openspace::properties::Property::PropertyInfo NumberPointsInfo = {
-        "AmountOfPoints",
+        "NumberOfPoints",
```

Comment:
Note that this is a breaking change.   It is not used anywhere in our assets though

Reply:
We should put it in the breaking change of the 0.20.0 though as some user might be using it.   I believe the Parameters struct had something like this as well?

Reply:
Yes, but that one is actually called `numberOfPoints`, so matches with the current renaming. 

Meaning, that this was also a bug fix